tests/TestCase/src/Controller/ProductsFrontendControllerTest.php

```diff
@@ -113,8 +114,8 @@ public function testProductDetailDeliveryBreakActive()
         $productId = 346;
         $manufacturerId = 5;
         $this->changeManufacturerNoDeliveryDays($manufacturerId, Configure::read('app.timeHelper')->getDeliveryDateByCurrentDayForDb());
-        $this->httpClient->get($this->Slug->getProductDetail($productId, 'Artischocke'));
-        $this->assertRegExpWithUnquotedString('<i class="fa fa-lg fa-times"></i> Lieferpause!', $this->httpClient->getContent());
```

Comment:
please use the built-in method "assertResponseContains" here (add backtick as delimiter to string)